spec/abstractops.html

```diff
@@ -95,32 +95,32 @@ <h1>GetStringOrNumberOption ( _options_, _property_, _stringValues_, _minimum_,
-    <h1>DurationHandleFractions ( _fHours_, _minutes_, _fMinutes_, _seconds_, _fSeconds_, _milliseconds_, _fMilliseconds_, _microseconds_, _fMicroseconds_, _nanoseconds_, _fNanoseconds_ )</h1>
+    <h1>DurationHandleFractions ( _fHours_, _minutes_, _fMinutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ )</h1>
     <emu-alg>
       1. If _fHours_ is not equal to 0, then
-        1. If any of _minutes_, _fMinutes_, _seconds_, _fSeconds_, _milliseconds_, _fMilliseconds_, _microseconds_, _fMicroseconds_, _nanoseconds_, _fNanoseconds_ is not 0, throw a *RangeError* exception.
+        1. If any of _minutes_, _fMinutes_, _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ is not 0, throw a *RangeError* exception.
         1. Let _mins_ be _fHours_ × 60.
         1. Set _minutes_ to floor(_mins_).
         1. Set _fMinutes_ to _mins_ modulo 1.
       1. If _fMinutes_ is not equal to 0, then
-        1. If any of _seconds_, _fSeconds_, _milliseconds_, _fMilliseconds_, _microseconds_, _fMicroseconds_, _nanoseconds_, _fNanoseconds_ is not 0, throw a *RangeError* exception.
+        1. If any of _seconds_, _milliseconds_, _microseconds_, _nanoseconds_ is not 0, throw a *RangeError* exception.
         1. Let _secs_ be _fMinutes_ × 60.
         1. Set _seconds_ to floor(_secs_).
         1. Set _fSeconds_ to _secs_ modulo 1.
-      1. If _fSeconds_ is not equal to 0, then
-        1. If any of _milliseconds_, _fMilliseconds_, _microseconds_, _fMicroseconds_, _nanoseconds_, _fNanoseconds_ is not 0, throw a *RangeError* exception.
-        1. Let _mils_ be _fSeconds_ × 1000.
-        1. Set _milliseconds_ floor(_mils_).
-        1. Set _fMilliseconds_ to _mils_ modulo 1.
-      1. If _fMilliseconds_ is not equal to 0, then
-        1. If any of _microseconds_, _fMicroseconds_, _nanoseconds_, _fNanoseconds_ is not 0, throw a *RangeError* exception.
-        1. Let _mics_ be _fMilliseconds_ × 1000.
-        1. Set _microseconds_ to floor(_mics_).
-        1. Set _fMicroseconds_ to _mils_ modulo 1.
-      1. If _fMicroseconds_ is not equal to 0, then
-        1. If either of _nanoseconds_, _fNanoseconds_ is not 0, throw a *RangeError* exception.
-        1. Let _nans_ be _fMicroseconds_ × 1000.
-        1. Set _nanoseconds_ to floor(_nans_).
+        1. If _fSeconds_ is not equal to 0, then
+          1. If any of _milliseconds_, _microseconds_, _nanoseconds_ is not 0, throw a *RangeError* exception.
+          1. Let _mils_ be _fSeconds_ × 1000.
+          1. Set _milliseconds_ floor(_mils_).
+          1. Set _fMilliseconds_ to _mils_ modulo 1.
+          1. If _fMilliseconds_ is not equal to 0, then
+            1. If either _microseconds_ or _nanoseconds_ is not 0, throw a *RangeError* exception.
+            1. Let _mics_ be _fMilliseconds_ × 1000.
+            1. Set _microseconds_ to floor(_mics_).
+            1. Set _fMicroseconds_ to _mils_ modulo 1.
```

Comment:
Shouldn't this be `_mics_` instead of `_mils_`? (I noticed this on the current version of the spec online and was going to open a PR for it, but I noticed this PR and figured I'd just make a comment here instead)

Reply:
yes, but that will be a Normative PR not an Editorial one.

Reply:
Ahh ok, thanks. I'm not too familiar with the tc39 process.

Reply:
no problem. Normative change mean the observed behavior will be changed by the change to the spec text (even if it is bug fix for an obvious mistake) which have much higher bar to be reviewed and changed post Stage 3. Editorial one - which is this particular PR , is purely internal reorganize the spec text in a way that no external observerable behavior will be changed, which have much lower bar to be reviewed and merged. I think you point out a very good one and we should fix it but just need to be done separately.

Reply:
separate issue filed for you in https://github.com/tc39/proposal-temporal/issues/1666